daemon/cmd/ipam.go

```diff
@@ -172,6 +172,18 @@ func (d *Daemon) DumpIPAM() *models.IPAMStatus {
 	return status
 }
 
+func (d *Daemon) allocateRouterIP(family datapath.NodeAddressingFamily) (net.IP, error) {
```

Comment:
How does this ensure that the `option.Config.LocalRouterIP` won't be allocated later on for a pod?

Reply:
This by itself won't prevent it from being clashed, the goal here is to provide a knob for users to specify a "good" value that it won't mess with the underlying networking.

Reply:
Would it be worthwhile to log a warning if the CIDR conflicts?